src/jit/gentree.cpp

```diff
@@ -10985,7 +10985,12 @@ void Compiler::gtDispTreeList(GenTreePtr tree, IndentStack* indentStack /* = nul
     }
 }
 
-// TODO(pdg): comment
+//------------------------------------------------------------------------
+// Compiler::gtDispRange: dumps a range of LIR to stdout.
+//
```

Comment:
for all your comments that mention "stdout" -- it's not really stdout, it's wherever jit dumps go. You don't need to say "to ...".